compose/compose-file/index.md

```diff
@@ -3,16 +3,15 @@ description: Compose file reference
 keywords: fig, composition, compose, docker
 redirect_from:
 - /compose/yml
-- /compose/compose-file-v3/
-title: Compose file version 3 reference
+title: Compose file
```

Comment:
Looks like, because the new file (`compose-file-v3`) was added in the same commit as this file was modified, we'll be losing git history (`git blame`) won't work for the new file; could you split this PR into two commits?

- first commit to rename `index.md` to `compose-file-v3.md`
- second commit to add `index.md` (with the content that's in this PR), and to update `compose-file-v3.md` if needed?

I think it's important to make sure that git blame still works, as sometimes there's specific history attached to changes.